infra/docker-compose/dev/knative/config-features.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: v1
```

Comment:
Rather than copy the file at `infra/cluster-init/knative-configmaps/config-features.yaml`, can we use a symlink ? May be it's hard to symlink it to work with docker, in which case we can make this the original and `infra/cluster-init/knative-configmaps/config-features.yaml` the symlink.

Reply:
Referenced `infra/cluster-init/knative-configmaps/config-features.yaml` instead!